SPECS/virtiofsd/virtiofsd.spec

```diff
@@ -50,6 +55,13 @@ install -D -p -m 0755 target/release/virtiofsd %{buildroot}%{_libexecdir}/virtio
 %{_libexecdir}/virtiofsd-rs
 
 %changelog
+* Fri Jan 26 2024 Nadiia Dubchak <ndubchak@microsoft.com> - 1.8.0-2
```

Comment:
I'm pretty sure these should be rolled into one changelog entry.

Reply:
Created a single changelog entry

Reply:
Created a single changelog entry